ce_comp.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python
+import os
+import sys
+import requests
+import json
+import argparse
+import ConfigParser
+from termcolor import colored, cprint
+from prettytable import PrettyTable
+import pandas as pd
```

Comment:
pandas dependency is inteded for csv export only or it is meant to be used in other parts of script's logic?

Reply:
Provide better error handling if no or wrong arguments are given.